IDE/Renesas/e2studio/RX72N/EnvisionKit/wolfssl/.cproject

```diff
@@ -118,6 +126,7 @@
 									<listOptionValue builtIn="false" value="&quot;.\wolfcrypt/src\pkcs7.obj&quot;"/>
 									<listOptionValue builtIn="false" value="&quot;.\wolfcrypt/src\poly1305.obj&quot;"/>
 									<listOptionValue builtIn="false" value="&quot;.\wolfcrypt/src\pwdbased.obj&quot;"/>
+									<listOptionValue builtIn="false" value="&quot;.\wolfcrypt/src\rabbit.obj&quot;"/>

```

Comment:
We have recently removed RABBIT from wolfSSL.  Can you rebase and remove rabbit.c references?

Reply:
Removed the reference.